planetary-social/nos
# Fix build in Xcode 16

## Issues covered
#1570 

## Description
Fixes the issues in Xcode 16:
- lots of warnings
- tab bar is broken on iPad/Mac
- SwiftUI previews

## How to test
1. run the app on iPad or Mac
2. make sure the tab bar looks good
3. try the previews
4. make sure there are no warnings
